sites/public/pages/account/edit.tsx

```diff
@@ -0,0 +1,323 @@
+    }
+  }
+
+  return (
+    <RequireLogin signInPath="/sign-in" signInMessage={t("t.loginIsRequired")}>
+      <FormsLayout>
+        <FormCard>
+          <div className="form-card__lead text-center border-b mx-0">
+            <Icon size="2xl" symbol="settings" />
+            <h2 className="form-card__title">{t("account.accountSettings")}</h2>
+          </div>
+          <SiteAlert type="notice" dismissable />
+          <Form id="update-name" onSubmit={handleSubmit(onNameSubmit)}>
+            {nameAlert && (
+              <AlertBox type={nameAlert.type} onClose={() => setNameAlert(null)} inverted closeable>
+                {nameAlert.message}
+              </AlertBox>
+            )}
+            <div className="form-card__group border-b">
+              <label className="field-label--caps" htmlFor="firstName">
+                {t("application.name.yourName")}
+              </label>
+
+              <Field
+                controlClassName="mt-2"
+                name="firstName"
+                placeholder={`${t("application.name.firstName")}`}
+                error={errors.firstName}
+                errorMessage={t("errors.firstNameError")}
+                register={register}
+                defaultValue={profile ? profile.firstName : null}
+              />
+
+              <Field
+                name="middleName"
+                placeholder={`${t("application.name.middleNameOptional")}`}
+                register={register}
+                defaultValue={profile ? profile?.middleName : null}
+              />
+
+              <Field
+                name="lastName"
+                placeholder={`${t("application.name.lastName")}`}
+                error={errors.lastName}
+                errorMessage={t("errors.lastNameError")}
+                register={register}
+                defaultValue={profile ? profile.lastName : null}
+              />
+              <div className="text-center">
+                <Button className="items-center">{t("account.settings.update")}</Button>
+              </div>
+            </div>
+          </Form>
+          <Form id="update-birthdate" onSubmit={handleSubmit(onBirthdateSubmit)}>
+            {dobAlert && (
+              <AlertBox type={dobAlert.type} onClose={() => setDobAlert(null)} inverted closeable>
+                {dobAlert.message}
+              </AlertBox>
+            )}
+            <div className="form-card__group border-b">
+              <DOBField
+                id="dateOfBirth"
+                name="dateOfBirth"
+                register={register}
+                error={errors?.dateOfBirth}
+                watch={watch}
+                defaultDOB={{
+                  birthDay: profile ? moment(new Date(profile.dob)).utc().format("DD") : null,
+                  birthMonth: profile ? moment(new Date(profile.dob)).utc().format("MM") : null,
+                  birthYear: profile ? moment(new Date(profile.dob)).utc().format("YYYY") : null,
```

Comment:
Ideally would be creating a helper function to do not repeat this 🐍 (not necessary of course!)